staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Expect.OverlayDualStack.conflist

```diff
@@ -0,0 +1,76 @@
+{
+  "cniVersion": "0.3.0",
+  "name": "azure",
+  "adapterName": "",
+  "plugins": [
+    {
+      "type": "azure-vnet",
+      "mode": "bridge",
+      "bridge": "azure0",
+      "executionMode": "v4swift",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "ipam": {
+        "type": "azure-cns",
+        "mode": "dualStackOverlay"
+      },
+      "dns": {
+        "Nameservers": [
+          "10.0.0.10",
+          "168.63.129.16"
+        ],
+        "Search": [
+          "svc.cluster.local"
+        ]
+      },
+      "AdditionalArgs": [
+        {
+          "Name": "EndpointPolicy",
+          "Value": {
+            "Type": "OutBoundNAT",
+            "ExceptionList": [
+              "10.240.0.0/16",
+              "fd12:3456::/64"
+            ]
+          }
+        },
+        {
+          "Name": "EndpointPolicy",
+          "Value": {
+            "Type": "ACL",
+            "Protocols": "6",
+            "Action": "Block",
+            "Direction": "Out",
+            "RemoteAddresses": "168.63.129.16/32",
```

Comment:
Does wireserver work with ipv6? We may need to block the ipv6 address of the wireserver if it works.

Reply:
there is no ipv6 IP for wireserver